keyboards/massdrop/alt/keymaps/zonfire99/rgb_matrix_kb.inc

```diff
@@ -0,0 +1,349 @@
+# copy to ../../../alt/rgb_matrix_kb.inc
+
+
+// !!! DO NOT ADD #pragma once !!! //
+
+// Step 1.
+// Declare custom effects using the RGB_MATRIX_EFFECT macro
+// (note the lack of semicolon after the macro!)
+RGB_MATRIX_EFFECT(RGB_CUSTOM_ZONFIRE)
+RGB_MATRIX_EFFECT(RGB_CUSTOM_ZONFIRE_2)
+RGB_MATRIX_EFFECT(RGB_CUSTOM_ZONFIRE_WHITE)
+//RGB_MATRIX_EFFECT(my_cool_effect2)
+
+ 
+// qmk_firmware # util/docker_build.sh massdrop/alt:default
+  
+
+// Step 2.
+// Define effects inside the `RGB_MATRIX_CUSTOM_EFFECT_IMPLS` ifdef block
+
+
+
+#ifdef RGB_MATRIX_CUSTOM_EFFECT_IMPLS
+
+// e.g: A simple effect, self-contained within a single method
+static bool RGB_CUSTOM_ZONFIRE(effect_params_t* params) {
+  RGB_MATRIX_USE_LIMITS(led_min, led_max);
+  for (uint8_t i = led_min; i < led_max; i++) {
```

Comment:
This is ... well, wrong.
 
You'd want to check to make sure that it's on the LED.  Or you should just use the `rgb_matrix_indicators_user` instead.